src/main/java/nl/ictm2a4/javagame/services/scores/ScoreService.java

```diff
@@ -0,0 +1,122 @@
+package nl.ictm2a4.javagame.services.scores;
```

Comment:
Wil je in deze Service, en eventueel andere, de functie namen met camelCasing schrijven?